elements/pfe-cta/src/pfe-cta.scss

```
@@ -70,95 +98,91 @@ $variables: (
   font-weight:     #{pfe-local(FontWeight)};
   line-height:     #{pfe-local($cssvar: LineHeight, $fallback: #{pfe-var(line-height)})};
   text-decoration: #{pfe-local(TextDecoration)} !important;
-  &::before {  // inner border on priority CTAs
-    content: "";
 
-    display:    block;
-    box-sizing: border-box;
-    position:   absolute;
-    top:    2px;
-    left:   2px;
-    bottom: 2px;
-    right:  2px;
+  :host(:not([pfe-priority],[aria-disabled="true"])) & {
+    padding-right: calc(#{pfe-local($cssvar: size, $region: arrow)} + #{pfe-local($cssvar: MarginLeft, $region: arrow)} + .375rem) !important;
+  }
 
-    border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} #{pfe-local($cssvar: BorderColor, $region: inner)};
-    border-radius: 2px;
-    outline: none;
+  :host(:hover) & {
+    color:   #{pfe-local(Color--hover)} !important;
+    text-decoration: #{pfe-local(TextDecoration--hover)} !important;
   }
 }
 
-::slotted(*:not(:disabled)) {
+:host(:not([aria-disabled="true"])) ::slotted(*:not(:disabled)),
+:host ::slotted(*:not(:disabled)) {
   cursor: pointer;
 }
 
-.pfe-cta--arrow {
-  display: #{pfe-local($cssvar: arrow--Display, $region: arrow)};
-  padding: #{pfe-local($cssvar: Padding, $region: arrow)};
-  fill:    #{pfe-local(Color)};
-  position: relative;
-  top:    .1rem;
-  width:  13px;
-  height: 13px;
-  transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+:host([aria-disabled="true"]) ::slotted(*) {
+  cursor: default !important;
 }
 
-  ::slotted(:hover) {
-    --pfe-cta__arrow--Padding:     0 0 0 .5rem;
-
-    color: #{pfe-local(Color--hover)} !important;
-    text-decoration: #{pfe-local(TextDecoration--hover)} !important;
-  }
-
-::slotted(:focus) {
-  outline: none !important;
-  color:   #{pfe-local(Color--focus)} !important;
-  text-decoration: #{pfe-local(TextDecoration--focus)} !important;
-
-  &::before {
-    border: 1px solid #{pfe-local($cssvar: BorderColor--focus, $region: inner)};
-  }
+.pfe-cta {
+  &--wrapper {
+    :host([pfe-priority]) &,
+    :host([aria-disabled="true"]) & {
+      display: flex;
+      flex-flow: row nowrap;
+      align-items: center;
+      height: 100%;      
+    }
 
-  .pfe-cta--arrow {
-    fill: #{pfe-local(Color--focus)};
+    @include browser-query(edge) {
+      button,
+      input {
+        @extend %reset-button;
+      }
+    }
   }
-}
-
-
+  &--inner {  // inner border on priority CTAs
+    display: block;
+    height: calc(100% - 4px);
+    width: calc(100% - 4px);
 
-:host(::after) {
-  margin-left: #{pfe-local($cssvar: MarginLeft, $region: arrow)};
-}
+    box-sizing: border-box;
+    position:   absolute;
+    top:    2px;
```

Moving my suggested code to improve arrow size & alignment in Edge:

```suggestion
  top:    -1px;
  vertical-align: middle;
  width: 0.9em;
  height: 0.9em;
```

---

This is the class for the inner border. Should this code go inside the arrow class instead?

---

@starryeyez024 bump ^

---

oh yes sorry, moved to the arrow